lib/routes/rides/routes.js

```diff
@@ -0,0 +1,118 @@
+import get from 'lodash/get';
+import { notFound, badImplementation, badRequest } from '@utils/responseInterceptors';
+import Joi from 'joi';
+import { bookARide, getPendingRides, assignDriverToRide } from '@daos/ridesDao';
+import { customerIdSchema } from '@utils/validationUtils';
+
+export default [
+  {
+    method: 'POST',
+    path: '/book/{customerId}',
+    options: {
+      description: 'Book a ride for a user',
+      notes: 'Book a ride for a user API',
+      tags: ['api', 'rides'],
+      validate: {
+        params: Joi.object({
+          customerId: customerIdSchema 
+        }),
+        payload: Joi.object({
+          pickUpLocation: Joi.string().trim().required().error(new Error('Pickup location is required.')),
+          dropLocation: Joi.string().trim().required().error(new Error('Drop location is required.'))
+        }),
+        options:{
+          allowUnknown: true,
+          abortEarly: true
+        },
+        failAction: async (request, h, err) => {
+          throw err;
+        }
+      },
+    },
+    handler: async (request, h) => {
+      const { payload, params } = request;
+      const { pickUpLocation, dropLocation } = payload;
+      const { customerId } = params;
+      return bookARide(pickUpLocation, dropLocation, customerId)
+        .then((ride) => h.response({
+          results: {
+            message: 'Ride is requested, please wait until we assign a driver to you.',
+            ride
+          },
+        }).code(201))
+        .catch((error) => badImplementation(error.message));
+    },
+  },
+  {
+    method: 'GET',
+    path: '/',
+    options: {
+      description: 'Get pending rides',
+      notes: 'Get pending rides API',
+      tags: ['api', 'rides'],
+      plugins: {
+        pagination: {
+          enabled: true,
+        },
+        query: {
+          pagination: true,
+        },
+      }
+    },
+    handler: async (request, h) => {
+      try {
+        const { query } = request;
+        const { limit, page } = query;
+        const { count, rows } = await getPendingRides(limit, page)
+          if(get(rows, 'length')) {
+            return h.response({
+              results: [
+                ...rows
+              ],
+              totalCount: count
+            })
+          }
+          return notFound('No Pending Rides found');
+      } catch (error) {
+        throw badImplementation(error.message)
+      }
+    },
+  },
+  {
+    method: 'POST',
+    path: '/accept/{driverId}/{rideId}',
```

Comment:
```suggestion
    path: '/{rideId}/accept',
```
Let's send `driverId` in body.